sourcegraph/about
# Neo Financial Case Study

This closes sourcegraph/sourcegraph#44677 and adds a new case study for Neo Financial.

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure links work in the case study
